ecf/include/envir-p1.h

```diff
@@ -15,3 +22,14 @@ fi
 export DBNROOT=$SIPHONROOT
 
 if [[ ! " prod para test " =~ " ${envir} " && " ops.prod ops.para " =~ " $(whoami) " ]]; then err_exit "ENVIR must be prod, para, or test [envir-p1.h]"; fi
+
+export COMROOT=/lfs/h2/emc/ptmp/%EMC_USER%/%PSLOT%/com
+export ROTDIR="$(compath.py -o prod/${NET}/${gfs_ver})"
+if [ -n "%PDY:%" ]; then
+  export PDY=${PDY:-%PDY:%}
+  export CDATE=${PDY}%CYC:%
+fi
+#### Required for archive job to do auto cleanup
+export DATAROOT=/lfs/h2/emc/stmp/%EMC_USER%/RUNDIRS/%PSLOT%/${CDATE}
+mkdir -p $DATAROOT
+if [ -n "%COMPATH:%" ]; then export COMPATH=${COMPATH:-%COMPATH:%}; fi
```

Comment:
This can be re-introduced/re-evaluated when an archive job is added.
Currently, there is no archive job being run.
Besides, this section seems very specific to the archive job and yet is being executed in `envir-p1.h` which is sourced by all jobs.

Reply:
Since they are not in the repository, I would prefer adding when the jobs are added.
Please reintroduce then.